util/Migrator/DbScripts/2023-01-20_00_OrganizationStatus.sql

```diff
@@ -0,0 +1,296 @@
+--Add column 'Status' to 'Organization' table
+IF COL_LENGTH('[dbo].[Organization]', 'Status') IS NULL
+BEGIN
+    ALTER TABLE
+        [dbo].[Organization]
+    ADD
+        [Status] SMALLINT NOT NULL CONSTRAINT [DF_Organization_Status] DEFAULT (0)
+END
+GO
+    
+--Updating existing Organizations to Status = Created
+UPDATE [dbo].[Organization]
+SET [Status] = 1
+GO
```

Comment:
I've set the column to be `NOT NULL` as I thought it would be strange to have any rows with an undefined value. I've changed the `default value = 1` that way all existing rows will have `Status = Created`. Only organizations created through the admin panel will have `Status = Pending`

Reply:
Looks good to me - that handles existing rows and the most common case (i.e. most orgs have `Created` status immediately, `Pending` is an exception).